### PR TITLE
15

### PR DESCRIPTION
**Pull request by ondrejbudai**
Release osbuild-composer version 15

### CODE REVIEW

**Comment by ondrejbudai** [comment]
I cherrypicked #627 and #767 and updated the NEWS. I consider this as 15-rc1. :-)

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+    s390x architectures.
+    
+  * Tar images for RHEL have returned. The Image Builder team found out that
+    they are used as a way to install RHEL for Satellite.
+
+  * Blueprints containing packages with a wildcard version no longer causes
+    the built image to have both x86_64 and i686 versions of one package
+    installed.
+  
+  * GPG check can now be disabled per repository. If you have a custom
```
Comment (teg) [inline NEWS.md:17]: It is opt in, default is not too check.

**Comment by ondrejbudai** [comment]
And we're ready!

**Comment by ondrejbudai** [comment]
Ouch, forgot to change the news, just a sec

**Comment by ondrejbudai** [comment]
aaand fixed!

File: NEWS.md (hunk 1 above)
Comment (ondrejbudai) [reply NEWS.md]: fixed

**Comment by teg** [comment]
The one test failure, is (obviously) unrelated to the release notes. It is a known problem under active investigation, though not reliably reproducible. It should not block the release.

**Comment by ondrejbudai** [comment]
I agree, let's do this!

**Pull request merged**